doc/odoc_for_authors.mld

```diff
@@ -487,15 +487,17 @@ the {{!page-driver}driver}.
 Tags are used to provide specific information for individual elements, such
 as author, version, parameters, etc. Tags start with an [@] symbol, appear
 at the end of documentation comments, and are not allowed elsewhere.
-They should appear on their own lines, with nothing but whitespace before them.
+They should appear on their own lines with nothing but whitespace before them.
 
-There are three types of tag: those with no associated data (simple tags),
-those with a single line of text (line tags) and those with a block of
+There are three types of tags. Those with:
+- no associated data (simple tags),
+- a single line of text (line tags), and 
+- a block of
 marked-up text (block tags).
 
 {3 Simple Tags}
 
-The three tags with no data are hints to the HTML renderer to do with [include]s.
+The three tags without data are hints to the HTML renderer to do with [include]s.
```

Comment:
```suggestion
The three tags without data are hints to the HTML renderer involving [include]s.
```

Is this still accurate?

Reply:
I think the intent is "it gives hints to the "renderer" on what to do with "include" elements.", So I say "involving" is not very accurate.